### DISCUSSION TITLE
PrismaAdapter schema issue

### DISCUSSION THREAD

**Discussion by r0ld3x** [nextauthjs/next-auth#10071] [Help] [answered]
> Here https://authjs.dev/reference/adapter/prisma#create-the-prisma-schema-from-scratch
```
model User {
  id            String    @id @default(cuid())
}
``` 

the id is string but in my database 

```
model User {
  id                Int                 @id @default(autoincrement())
``` 
its integer 

if I change String   to Int prisma Adpater isnot working and next-auth is showing error which is getting fixed when i am using id  string

so is there any way to fix this?

**Comment by luclissi** [comment, marked as answer]
Related [issue](https://github.com/nextauthjs/next-auth/issues/4115) [discussion](https://github.com/nextauthjs/next-auth/discussions/9776)

**Reply by r0ld3x** [reply]
thanks :)
